packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testBrowseInstanceMethod.st

```diff
@@ -0,0 +1,14 @@
+tests
+testBrowseInstanceMethod
+	
+	| browser tree |
+	
+	tree := IPTTreeModelMock new 
+		roots: {tallyObject};
+		currentSelection: tallyObject.
+		
+	browser := tree browseMethodCall.
+	
+	[self assert: browser selectedMessageName = #setUp.
```

Comment:
I’d recommend using the shorthand `assert:equals:` here (and in the other assertions).